src/20_Components/amp-analytics.html

```diff
@@ -62,15 +61,15 @@
       background: green;
       color: var(--color-text-light);
     }
-    form #testForm {
+    #testForm {
       padding: var(--space-2);
     }
-    form #testForm div {
+    #testForm div {
       margin-bottom: var(--space-3);
       position: relative;
       display: inline-block
     }
-    button, form #testForm input {
+    button, #testForm input[type=submit] {
       text-transform: uppercase;
       letter-spacing: .2em;
       font-family: inherit;
```

Comment:
I'd suggest removing:

```
text-transform: uppercase;
letter-spacing: .2em;
font-family: inherit;
```

as these are not needed.

Reply:
...basically all custom styles (except for the dashboard which should look good).

Reply:
removed